lib/svgo/coa.js

```diff
@@ -131,6 +139,7 @@ module.exports = require('coa').Cmd()
         .end()
     .act(function(opts, args) {
 
+	    console.log(`opts - ${opts.folder}, args - ${args}`)
```

Comment:
Has it left from debugging? SVGO works on node 0.12 and higher, where is no support for template strings.